Fix column command calls

e-n-l opened the pull request:
The current script returns the following message on execute:
```
column: option requires an argument -- 'n'
Try 'column --help' for more information.
column: option requires an argument -- 'n'
Try 'column --help' for more information.
```

This update supplies `column` with arguments sufficient to return the expected output

Commit:
dev-ls.plx - avoid stty command warning

Commit:
Merge branch 'master' of https://github.com/e-n-l/devicons-shell

Commit:
Rewrite devicons in perl for consistent results

Commit:
devicons-ls - fix column command usage
 '-n' modifier is not applicable

ryanoasis commented:
@e-n-l Thanks for this :+1: but honestly I am confused. What is the difference as a result of this versus not even having the '-n' option? For me there is no visual difference. Can you show an example of the "-s"$symbol" -t " vs. no option?

Also to be perfectly honest I don't know why that is needed to delimit? From what I see in the code the $symbol is only used internally by the functions :thinking:

e-n-l commented:
TBH, I'm pretty new to bash... I was just trying to get it to behave based on what I'd read on the `column` man page, got something working and committed it without fully grokking what it was doing:

![image](https://user-images.githubusercontent.com/487308/39100694-a894588e-4642-11e8-94ef-42c77caa3bd4.png)

I've been trying to add color, based on better-ls, but it's screwed up the columns for some reason:

![image](https://user-images.githubusercontent.com/487308/39100712-d1b1f6c2-4642-11e8-9128-a79819787382.png)

ryanoasis commented:
Ah okay no worries.

Just wondering if removing the -n flag was enough or you had to add the flags you put in your pull request? I.e. did the coloring you are doing only working with the PR you submitted?

Would you mind sharing your code for color that you are doing or at least some of it? If you want that is  :smiley:

Commit:
WIP - colorize extensions, add .7z example

e-n-l commented:
So the top clip is what I'd submitted; seems to be working pretty well / consistently, but no colors. 

The bottom is the [WIP here](https://github.com/e-n-l/devicons-shell/blob/master/devicons-ls), which I started doing when installing [even-better-ls](https://github.com/illinoisjackson/even-better-ls) didn't work.

e-n-l committed:
Fix lisp icon

Lambda character causes problems, for some reason... reverting to default src char. 
Remove dependency on Data::Dump

e-n-l commented:
Okay, I really wanted this to work, so I just re-wrote the whole thing in Perl. I'm not sure if this even makes sense as a PR, or if I should make this a separate fork ala better-ls.

In any case, it works, it's pretty, and it's MUCH better organized for easy customization of icons and colors.

ryanoasis commented:
> Okay, I really wanted this to work, so I just re-wrote the whole thing in Perl

It looks great! I have honestly never used Perl but it wasn't too bad to figure out what you were doing and I understand the basic logic.

> I'm not sure if this even makes sense as a PR, or if I should make this a separate fork ala better-ls.

So I was actually going back and forth on this. I first thought probably fork but I like the changes and the bash is still the basic 'glue' for all the logic. I think even calling it 'devicons-shell' even still follows though maybe there is an even better generic name.

Anyway, I'd like to merge this. I saw a problem with the alignment though:

![selection_022](https://user-images.githubusercontent.com/8083459/39408961-669137e4-4bac-11e8-82f8-096b1f802499.png)

I could not quickly figure it out but somehow changing the echo to this: `echo -e "$lines \n" | column -t | column` worked for me:

![selection_023](https://user-images.githubusercontent.com/8083459/39408999-f7098330-4bac-11e8-91d7-1af67aa7fdb9.png)
